nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -135,8 +178,20 @@ def minhash32(
         """
         if not isinstance(ser, cudf.Series):
             raise TypeError("Expected data of type cudf.Series")
-        seeds = cudf.Series(seeds, dtype="uint32")
-        return ser.str.minhash(seeds=seeds, width=char_ngram)
+
+        if not MINHASH_PERMUTED_AVAILABLE:
+            self._logger.warning(
+                "Using an older implementation of minhash, update to cudf >= 24.12"
```

Comment:
```suggestion
                "Using an outdated minhash implementation, please update to cuDF version 24.12 or later for improved performance."
```

Reply:
Does a `FutureWarning` or something similar make sense? Maybe not